packages/express/src/middleware/index.ts

```diff
@@ -34,8 +34,18 @@ export function expressMiddleware(appsignal: Client): RequestHandler {
       res.end = function (this: Response, ...args: any) {
         res.end = originalEnd
 
-        const { method = "GET", params = {}, query = {}, headers } = req
+        const {
+          method = "GET",
+          params = {},
+          query = {},
+          body = {},
+          headers
+        } = req
         const filteredHeaders = filterHeaders(headers, appsignal)
+        const objectBody =
+          body instanceof Object && Object.keys(body).length > 0
```

Comment:
This could also check that `!(body instanceof Buffer)`, to avoid sending the raw body bytes if the Express application uses [`express.raw()`](http://expressjs.com/en/api.html#express.raw).

Reply:
Probably a good idea to check it 👍 . If it's a Buffer the `Object.keys` function will return length > 0, but the keys are numbers (indexes).

Reply:
Thanks for noticing! Feedback applied.